packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -742,6 +767,22 @@ export class Project extends ProjectBase {
     this._secondaryArtifacts.push(secondaryArtifact.bind(this, this).artifactsProperty);
   }
 
+  public bindToCodePipeline(_scope: Construct, options: BindToCodePipelineOptions): void {
+    // work around a bug in CodeBuild: it ignores the KMS key set on the pipeline,
```

Comment:
Can you validate here that this project uses pipeline source?

Reply:
Sure.